README.md

```diff
@@ -1,17 +1,44 @@
 # Report Service
-Ingest student runs and activity structure in order to run report queries later.
 
-## Scripts
+This repo contains a seriers of related projects that enable researchers to run reports on portal learner data.
+
+A detailed description of the system can be found at https://docs.google.com/document/d/1F4ozfQOjzZfMMYdGiPp0MGWpAxjTbMa6lDIvPSnuNcI/
+
+The high-level parts:
+
+* **Scripts**: Pushes old learner data from Firestore into S3, for access by Athena. See
+    [scripts/README.md](scripts/README.md)
+* **Functions**: Keeps new learner data synced to S3, as well as activity structures. See
+     [functions/README.md](functions/README.md)
```

Comment:
This also provides the API used LARA to send learner data and activity structure to FireStore. And a API used by the portal to move learner data from one class to another.